api_app/script_analyzers/observable_analyzers/dnstwist.py

```diff
@@ -0,0 +1,58 @@
+import subprocess
+import json
+import logging
+from shutil import which
+
+from api_app.exceptions import AnalyzerRunException
+from api_app.script_analyzers import classes
+
+
+class DNStwist(classes.ObservableAnalyzer):
+    command: str = "dnstwist"
+    dictionary_base_path: str = "/opt/deploy/dnstwist-dictionaries/"
+    final_report = {}
```

Comment:
care, this is a variable of the class. I would put this directly inside the `run` function.

Reply:
Oh okay sure. May I ask why ? I don't think I know when to make a variable of the class and when not to.

Reply:
in this way, that variable is shared among all the instances of the DNStwist class. The report must be different between each execution, so it can't be here. (https://www.digitalocean.com/community/tutorials/understanding-class-and-instance-variables-in-python-3). Otherise the other strings you put are correctly placed there because they are the same in each execution.